Xamarin.Forms.Platform.Android/ViewExtensions.cs

```diff
@@ -116,8 +116,7 @@ internal static void MaybeRequestLayout(this AView view)
 		internal static T GetParentOfType<T>(this IViewParent view)
 			where T : class
 		{
-			T t = view as T;
-			if (view != null)
+			if (view != null && view is T t)
```

Comment:
```suggestion
			if (view is T t)
```

`is T t` already checks for null (but `is var t` is always true...